lib/main.dart

```diff
@@ -108,6 +118,23 @@ class _RandomWordsState extends State<RandomWords> {
   }
 }
 
+class Info extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text("Info"),
+      ),
+      body: const Center(
+          child: Text(
+        "Here you can add your words to the cart. \nYes!!! You can buy them. \n The future is now, wake up",
```

Comment:
I can't understand the purpose of this page. You can add words to the cart on the first (main) page, but you claim to do the same thing on the info page. 
The idea is cool but needs some tuning.

P.S. It would be better to save your text to the constant and then use that constant.

Reply:
I think the purpose of this page is to inform users what is on the 'Saved Suggestions' page but not to add words.

Reply:
In another pull of requests, I expanded the functionality of the application

Reply:
I wouldn't recommend splitting a single feature into multiple PRs in case this feature is small (like yours).